.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
We need to check how this works out with us using GitHub-Projects.
On the contrary defining a project-task template might not be the worst of ideas.